code/MemberImportController.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
This class, as well as the `MemberImportForm` and `GroupImportForm` classes are obsolete.
Remove these classes and their corresponding javascript/styles. Add the following to `SecurityAdmin` instead:
```php
    private static $model_importers = [
       'users' => MemberCsvBulkLoader::class,
       Member::class => MemberCsvBulkLoader::class,
       'groups' => GroupCsvBulkLoader::class,
       Group::class => GroupCsvBulkLoader::class,
    ];
```
Note: We have to add both the model tab reference _and_ the class name as keys for the importers because `ModelAdmin` currently checks for `$importers[$modelClass]` in some places and `$importers[$this->modelTab]` in others. This is a bug that we should fix in CMS 5 but up to you if it's done separately or with this PR. If you want this to be handled separately let me know and I'll create a new card for it.

Then in `getGridFieldConfig()` remove the `setImportIframe()` calls and reference to `MemberImportController`.

Finally, if you want to include the advanced usage details from the old `MemberImportForm` and `GroupImportForm` classes (e.g "Groups can be assigned by the <em>Groups</em> column. Groups are identified by their <em>Code</em> property multiple groups can be separated by comma. Existing group memberships are not cleared." etc) you can override `ImportForm()` and add them to the form that `ModelAdmin` already creates. You'll also need to add `ImportForm` as an allowed action if you do that since allowed actions are only inherited for methods that aren't overridden:
```php
    private static $allowed_actions = [
        'ImportForm',
    ];
```

Reply:
> If you want this to be handled separately let me know and I'll create a new card for it.

New card would be great thanks.  I'll add an inline comment in here for now explaining why there's duplicates

Reply:
https://github.com/silverstripe/silverstripe-admin/issues/1364